nginx-autoinstall.sh

```diff
@@ -8,8 +8,8 @@ fi
 # Variables
 NGINX_MAINLINE_VER=1.15.8
 NGINX_STABLE_VER=1.14.2
-LIBRESSL_VER=2.7.5
-OPENSSL_VER=1.1.1
+LIBRESSL_VER=2.8.3
```

Comment:
LibreSSL 2.9.0 is out : https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.9.0-relnotes.txt

Reply:
Thank you for the suggestion, however this is just a development release.

Reply:
LibreSSL 2.9.2 is now ready for production usage https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.9.2-relnotes.txt